internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1369,27 +1384,36 @@ func configAdvanced(projectID, clusterName string, p *admin20240530.ClusterDescr
 			project_id = mongodbatlas_advanced_cluster.test.project_id
```

Comment:
Just to a double check from our draft checklist, is the attribute supported in older MongoDB versions? Could be good to try an older MongoDB version and attribute is not defined.

Reply:
added validation and acceptance test for this scenario